core/field_image.js

```diff
@@ -166,28 +167,28 @@ Blockly.FieldImage.prototype.setTooltip = function(newTip) {
 };
 
 /**
- * Get the source URL of this image.
- * @return {string} Current text.
- * @override
+ * Ensure that the input value is a string.
```

Comment:
Ensure that the input value _(the source URL)_ is a string.

Reply:
Fixed